### ISSUE TITLE
Build failed on macOS with QT 5.9

### ISSUE THREAD

**Issue by kpy3** [KDAB/GammaRay#374]
```
% brew install -vvv gammaray-qt5
==> Installing gammaray-qt5 from kdab/tap
==> Installing dependencies for kdab/tap/gammaray-qt5: kdstatemachineeditor-qt5
==> Installing kdab/tap/gammaray-qt5 dependency: kdstatemachineeditor-qt5
==> Downloading https://github.com/KDAB/KDStateMachineEditor/archive/v1.2.0.tar.gz
Already downloaded: /Users/yes/Library/Caches/Homebrew/kdstatemachineeditor-qt5-1.2.0.tar.gz
==> Verifying kdstatemachineeditor-qt5-1.2.0.tar.gz checksum
tar xzf /Users/yes/Library/Caches/Homebrew/kdstatemachineeditor-qt5-1.2.0.tar.gz
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/kdstatemachineeditor-qt5/1.2.0 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev
-- The C compiler identification is AppleClang 8.1.0.8020042
-- The CXX compiler identification is AppleClang 8.1.0.8020042
-- Check for working C compiler: /usr/local/Homebrew/Library/Homebrew/shims/super/clang
-- Check for working C compiler: /usr/local/Homebrew/Library/Homebrew/shims/super/clang -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/local/Homebrew/Library/Homebrew/shims/super/clang++
-- Check for working CXX compiler: /usr/local/Homebrew/Library/Homebrew/shims/super/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for include file graphviz/graphviz_version.h
-- Looking for include file graphviz/graphviz_version.h - found
-- Found Graphviz version 2.40.1
-- Looking for include file string.h
-- Looking for include file string.h - found
-- Found Graphviz: /usr/local/lib/libcdt.dylib /usr/local/lib/libgvc.dylib /usr/local/lib/libcgraph.dylib GRAPHVIZ_GRAPH_LIBRARY-NOTFOUND /usr/local/lib/libpathplan.dylib
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Performing Test C_SUPPORTS_UNUSED_BUT_SET
-- Performing Test C_SUPPORTS_UNUSED_BUT_SET - Success
-- Performing Test CXX_SUPPORTS_UNUSED_BUT_SET
-- Performing Test CXX_SUPPORTS_UNUSED_BUT_SET - Success
-- Performing Test C_SUPPORTS_LOGICAL_OP
-- Performing Test C_SUPPORTS_LOGICAL_OP - Success
-- Performing Test CXX_SUPPORTS_LOGICAL_OP
-- Performing Test CXX_SUPPORTS_LOGICAL_OP - Success
-- Performing Test C_SUPPORTS_POINTER_MEMACCESS
-- Performing Test C_SUPPORTS_POINTER_MEMACCESS - Success
-- Performing Test CXX_SUPPORTS_POINTER_MEMACCESS
-- Performing Test CXX_SUPPORTS_POINTER_MEMACCESS - Success
-- Performing Test C_SUPPORTS_REORDER
-- Performing Test C_SUPPORTS_REORDER - Success
-- Performing Test CXX_SUPPORTS_REORDER
-- Performing Test CXX_SUPPORTS_REORDER - Success
-- Performing Test C_SUPPORTS_FORMAT_SECURITY
-- Performing Test C_SUPPORTS_FORMAT_SECURITY - Success
-- Performing Test CXX_SUPPORTS_FORMAT_SECURITY
-- Performing Test CXX_SUPPORTS_FORMAT_SECURITY - Success
-- Performing Test HAVE_GXX_GNUXX11
-- Performing Test HAVE_GXX_GNUXX11 - Success
-- Performing Test HAVE_GXX_CXX11
-- Performing Test HAVE_GXX_CXX11 - Success
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE)
-- Enabling use of experimental 'cgraph' library of GraphViz
-- The following OPTIONAL packages have been found:

 * Qt5RemoteObjects (required version >= 5.4), Qt Remote Objects module
   Needed for the QStateMachine/QtSCXML adapter and remote debugging capabilities
 * Qt5Scxml (required version >= 5.8), Qt SCXML module
   Needed for the Qt SCXML adapter (adapter itself depends on Qt5RemoteObjects)
 * Qt5XmlPatterns, Qt5 XmlPatterns library
   Required for unit tests dealing with XML input/output

-- The following RECOMMENDED packages have been found:

 * Graphviz, Graph visualization software, <http://www.graphviz.org/>
   Needed for automatic layout of state charts

-- The following REQUIRED packages have been found:

 * Qt5Core (required version >= 5.9.0)
 * Qt5Gui (required version >= 5.9.0)
 * Qt5Network (required version >= 5.9.0)
 * Qt5Qml (required version >= 5.9.0)
 * Qt5Quick
 * Qt5Test
 * Qt5Widgets
 * Qt5QuickWidgets
 * Qt5 (required version >= 5.3)

-- The following features have been disabled:

 * Internal build of Graphviz, enable with WITH_INTERNAL_GRAPHVIZ=ON

-- The following OPTIONAL packages have not been found:

 * Doxygen, Doxygen documentation generator
   Needed for generating API documentation (make docs)

-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/kdstatemachineeditor-qt5-20170613-42830-nqammj/KDStateMachineEditor-1.2.0
==> make

....


[ 55%] Built target kdstatemachineeditor_view
make: *** [all] Error 2
```

==> Formula
```
Tap: kdab/tap
Path: /usr/local/Homebrew/Library/Taps/kdab/homebrew-tap/kdstatemachineeditor-qt5.rb
==> Configuration
HOMEBREW_VERSION: 1.2.3
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: ed9942fbd1ec4243f0a92ab8f9b2411c8b1fb091
Last commit: 16 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: c88f72186513b4d183b79def32f52ea13212de43
Core tap last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit broadwell
Homebrew Ruby: 2.0.0-p648
Clang: 8.1 build 802
Git: 2.13.1 => /usr/local/bin/git
Perl: /usr/local/bin/perl => /usr/local/Cellar/perl/5.24.1/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_112, 1.8.0_60
macOS: 10.12.5-x86_64
Xcode: 8.3.3
CLT: 8.3.2.0.1.1492020469
X11: 2.7.9 => /opt/X11
==> ENV
HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
SDKROOT: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk
MAKEFLAGS: -j4
CMAKE_PREFIX_PATH: /usr/local/opt/qt:/usr/local
CMAKE_INCLUDE_PATH: /usr/include/libxml2:/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
CMAKE_FRAMEWORK_PATH: /usr/local/opt/qt/Frameworks
MACOSX_DEPLOYMENT_TARGET: 10.12
PKG_CONFIG_PATH: /usr/local/opt/qt/lib/pkgconfig:/usr/local/opt/libpng/lib/pkgconfig:/usr/local/opt/freetype/lib/pkgconfig:/usr/local/opt/fontconfig/lib/pkgconfig:/usr/local/opt/libtiff/lib/pkgconfig:/usr/local/opt/webp/lib/pkgconfig:/usr/local/opt/gd/lib/pkgconfig:/usr/local/opt/graphviz/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig/10.12
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/Homebrew/Library/Homebrew/shims/super:/usr/local/opt/qt/bin:/usr/local/opt/libtool/bin:/usr/local/opt/libpng/bin:/usr/local/opt/freetype/bin:/usr/local/opt/fontconfig/bin:/usr/local/opt/jpeg/bin:/usr/local/opt/libtiff/bin:/usr/local/opt/webp/bin:/usr/local/opt/gd/bin:/usr/local/opt/graphviz/bin:/usr/local/opt/cmake/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

**Comment by winterz** [comment]
looks like I need to update the kdstatemachineeditor package too.

**Comment by winterz** [comment]
verified.  we need to release a kdstatemachineeditor 1.2.1 which I'll try to get done within a few days.

**Comment by winterz** [comment]
I just released KDStateMachineEditor 1.2.1 and updated the homebrew formula.
brew install gammaray-qt5 works now.
